Packs/Viper/Integrations/Viper/Viper.py

```diff
@@ -0,0 +1,181 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' IMPORTS '''
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class ViperClient(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def sample_information(self, file_hash):
+        '''Get Sample instance information from Viper'''
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/malware/{file_hash}/'
+        )
+
+    def test_module(self):
+        return self._http_request(method='GET', url_suffix='/')
+
+    def sample_download(self, file_hash):
+        '''Download Sample from Viper'''
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/malware/{file_hash}/download'
+        )
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+
+    Args:
+        client: Viper client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test
+    """
+    client.test_module()
+    return 'ok'
+
+
+def sample_download(file_hash):
```

Comment:
Seems that its a duplicate of your client function.
I would recommend to remove the unnecessary code

Reply:
the sample download is needed to fetch the file.

Reply:
right, however you have implemented this function as part as you ViperClient already.
As far as I can see you are using you client sample_download function later in the code.
So this is not actually used.